TODO.md

```diff
@@ -38,6 +39,9 @@
   - linux
     - clear ff history?
 
+- cool/stretch goal
```

Comment:
We could make `data-binary` on `.binary` initially their ID

Reply:
The `mc.TeamID` field is refreshed every cycle, so we could just add it to `genReport()` I think.